File: codility/MaxNonoverlappingSegments.js
function solution(A, B) {
    // Implement your solution here
    let counter = 0;
    let min = -1;
    for (let i = 0; i < A.length; i++) {
        if (A[i] > min) {
            counter++;
            min = B[i];
        }
    }
    return counter;
}

File: codility/python/MaxNonoverlappingSegments.py
def solution(A, B):
    # Implement your solution here
    m = -1
    count = 0
    n = len(A)
    for i in range(n):
        a = A[i]
        b = B[i]
        if a > m:
            m = b
            count += 1
    return count